README.md

```diff
@@ -20,7 +20,7 @@ should return the user instances who have recipe cards with this recipe
 - `Recipe#ingredients`
 should return all of the ingredients in this recipe
 - `Recipe#allergens`
-should return all of the ingredients in this recipe that are allergens
+should return all of the ingredients in this recipe that users are allergic to
```

Comment:
"...`Ingredient`s in this recipe that are allergens for `User`s in our system"

This hints that the join is the key of Allergen to User via Allergy.